### DISCUSSION TITLE
Disable GPU in Docker

### DISCUSSION THREAD

**Discussion by Fx4-X** [mindsdb/mindsdb#1395] [Get help] [answered]
Hello,

I try to install mindsdb using instruction from https://docs.mindsdb.com/installation/docker/
docker logs through below error, and yes there are no GPU on the docker host.
and the URL is not opening and send 4xx error.

How can be disable the GPU in docker or avoid the error ?

`/opt/conda/lib/python3.8/site-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /opt/conda/conda-bld/pytorch_1603729096996/work/c10/cuda/CUDAFunctions.cpp:100.)
  return torch._C._cuda_getDeviceCount() > 0
/opt/conda/lib/python3.8/site-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /opt/conda/conda-bld/pytorch_1603729096996/work/c10/cuda/CUDAFunctions.cpp:100.)
  return torch._C._cuda_getDeviceCount() > 0
/opt/conda/lib/python3.8/site-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /opt/conda/conda-bld/pytorch_1603729096996/work/c10/cuda/CUDAFunctions.cpp:100.)
`

**Comment by ZoranPandovski** [comment, marked as answer]
Hello @Fx4-X this is strange. So, you don't have a GPU on the host machine but MindsDB image is still defaulting to the use GPU option. Let us investigate this and I will be back with more info. Just to confirm, you got this error on mindsdb start or when you start with the model training?

**Reply by Fx4-X** [reply]
Hello @ZoranPandovski ,

Looks like the issue resolved in the "latest" docker image which released on 27th Aug 2021.
now its showing the warning and proceeding, which is okay. 
`  warnings.warn("torch.cuda.amp.GradScaler is enabled, but CUDA is not available.  Disabling.")
`

However the issue was at docker tag v2.15.0 or latest at the time of this GH issue creation, its no more relevant now.

Different question; the docker tag "latest" have the mindsdb version 2.45.2, why there are no image with version tag ? ex : v2.45.2

**Reply by ZoranPandovski** [reply]
The latest docker image is always the version of the latest release tag from GitHub. This is a  good question, maybe we should consider tagging the images.

**Reply by Fx4-X** [reply]
Not really, the mindsdb images at docker hub is not get updated regularly. It does happen several months apart, and with many version gap.
